core/src/main/scala/kafka/tools/StreamsResetter.java

```diff
@@ -138,12 +138,12 @@ private void parseArguments(final String[] args) throws IOException {
             .ofType(String.class)
             .defaultsTo("localhost:2181")
             .describedAs("url");
-        inputTopicsOption = optionParser.accepts("input-topics", "Comma-separated list of user input topics")
+        inputTopicsOption = optionParser.accepts("input-topics", "Comma-separated list of user input topics. For these topics, the tool will reset the offset to 0.")
```

Comment:
Nit: I think this is not always true -- it will reset to the earliest available offset, which may not necessarily be `0`, right?  (same comment applies to the other mention of `offset 0` in the CLI help output below).